src/playlist-maker.js

```diff
@@ -604,18 +596,20 @@ export default function factory(player, initialList, initialIndex = 0) {
   /**
    * Plays the next item in the playlist.
    *
+   * @param {boolean} [suppressPoster]
+   *         Should the native poster be suppressed? Defaults to false.
    * @return {Object|undefined}
    *         Returns undefined and has no side effects if on last item.
    */
-  playlist.next = () => {
+  playlist.next = (suppressPoster = false) => {
```

Comment:
I'd be okay with that though it should be consistently applied in the other two cases above. I'm fine with this as-is, though. Either way works for me (especially for something that's only around for the life of 5.x).

Reply:
I am okay with both approaches here,
it is just a kind / "clean code" reminder :)

Reply:
I considered this but for consistency it should be everywhere and that's a breaking change because it changes the signature of playlist.currentItem (and playItem, but I don't think that's documented as part of the api)

Also - just to note that bouncing about between branches I somehow missed off the most important change in playItem! That's been added back in with the last commit.

Reply:
I got it, thank you for this clarification!